docs/data_dictionaries/index.rst

```diff
@@ -3,20 +3,40 @@
 Data Dictionaries
 =================
 
+Data Processed & Cleaned by PUDL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The PUDL data dictionary provides detailed metadata for the tables
+in the PUDL database. This includes table descriptions,
+field names, field, descriptions, and field datatypes.
```

Comment:
Is the comma between field and descriptions is a typo?

Reply:
Yes, that was a mistake. It has been resolved